components/input.css

```diff
@@ -0,0 +1,39 @@
+.input_main {
+  border: 2px solid var(--gray-border);
+  border-radius: 5px;
+  margin: 1rem;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 1rem;
+}
+
+.text-label {
+  margin: 1rem;
+  margin-bottom: 0;
+  font-size: 1rem;
+  line-height: 1;
+  font-weight: 400;
+}
+
+.textbox {
+  padding: 0.5rem 0.5rem;
```

Comment:
You can also do `padding: 0.5rem;` here as all four sides have same padding

Reply:
yes.